KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/ILiveManager.java

```diff
@@ -40,4 +40,6 @@
 	public KalturaLiveAsset getLiveAsset(String entryId, int assetParamsId);
 	
 	public KalturaLiveParams getLiveAssetParams(int assetParamsId);
+	
+	public Object getMetadata(String entryId, String key, Object defaultValue);
```

Comment:
I think that the right place for this metadata would have been on the KalturaLiveEntry object. However, since this API is tightly coupled to the client API and we don't want to change it drastically (until the refactor) that's not an option.
Either way, as we discussed, this API doesn't make sense to me.
How about: public Map<String,Object> getMetadata(String entryId) which always returns a non-null dictionary?
Then it's up to the client to set/get stuff from it.